crypto.go

```diff
@@ -546,8 +546,10 @@ const (
 //    opaque label<9..255>;
 //    opaque hash_value<0..255>;
 // };
+var HkdfLabelPrefix = "tls13 "
```

Comment:
`HKDFLabelPrefix`

Reply:
Is the idea that if you were doing QUIC or something, you would do something like `mint.HKDFLabelPrefix = "quic "`?  Does that actually work?

Reply:
That wouldn't work because you only want to use the "quic " label in certain limited cases.  The rest of the key schedule keeps the "tls13 " label.  It's stupendously inconsistent.  For which I blame ekr.